src/Checkout/Helpers/ReserveStock.php

```diff
@@ -36,6 +36,24 @@ protected function is_enabled() {
 		return $this->enabled;
 	}
 
+	/**
+	 * Returns the amount of minutes to use for reserving stock stock.
+	 *
+	 * @param  integer $minutes amount of minutes to reserve stock.
+	 * @return integer $minutes filtered minutes.
+	 */
+	public function reservation_minutes( $minutes = 0 ) {
+
+		$minutes = $minutes ? $minutes : (int) get_option( 'woocommerce_hold_stock_minutes', 60 );
```

Comment:
I wonder if we should consider deprecating the use of this option internally and require the `$minutes` parameter? That could arguably be a breaking change and I think we will just be stuck with it.